docs/UserGuide.md

```diff
@@ -6,110 +6,131 @@
 
 # Dormie User Guide
 
-Dormie is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Dormie can get your contact management tasks done faster than traditional GUI apps.
+Dormie is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Dormie can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
Should CLI be "Command Line Interface" instead?

Reply:
updatedd